frontend/components/home/HAboutUs.tsx

```
@@ -0,0 +1,135 @@
+                                        <DialogTitle className='hidden' />
+                                        <DialogDescription className='hidden' />
+                                    </DialogHeader>
+                                    
+                                    <iframe
+                                        src='http://youtube.com/embed/zpOULjyy-n8?rel=0'
+                                        title='youtube video'
+                                        className='w-full h-48 md:h-52 lg:h-56 xl:h-60'
+                                        frameBorder='0'
+                                        allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share'
+                                        allowFullScreen
+                                    ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)